### PR TITLE
inline and eliminate get_authenticated_api_client() shortcut

### PR DESCRIPTION
**Pull request by das-g**
With login automated in 8d86653 and 8a479a4, it wasn't much of a shortcut anymore.

#### Reviewed by:
- [x] @hixi

### CODE REVIEW

File: osmaxx/excerptexport/forms/existing_form.py [hunk 1]
```diff
@@ -81,10 +81,10 @@ def save(self, user):
 
         existing_key = self.cleaned_data['existing_excerpts']
         if self._is_country(existing_key):
-            from osmaxx.api_client.shortcuts import get_authenticated_api_client
+            from osmaxx.api_client import ConversionApiClient
             country_id = int(existing_key.strip(COUNTRY_ID_PREFIX))
             extraction_order.country_id = country_id
-            extraction_order.name = get_authenticated_api_client().get_country_name(country_id)
+            extraction_order.name = ConversionApiClient().get_country_name(country_id)
```
Comment (hixi) [inline osmaxx/excerptexport/forms/existing_form.py]: Are you aware, that  countries are not on the ConversionService anymore?

File: osmaxx/excerptexport/models/extraction_order.py [hunk 2]
```diff
@@ -84,8 +84,8 @@ def excerpt_name(self):
         if self.excerpt:
             return self.excerpt.name
         elif self.country_id:
-            from osmaxx.api_client.shortcuts import get_authenticated_api_client
-            return get_authenticated_api_client().get_country_name(self.country_id)
+            from osmaxx.api_client import ConversionApiClient
+            return ConversionApiClient().get_country_name(self.country_id)
```
Comment (hixi) [inline osmaxx/excerptexport/models/extraction_order.py]: Are you aware, that countries are not on the ConversionService anymore?

**Comment by hixi** [comment]
:+1: for the removal

File: osmaxx/excerptexport/forms/existing_form.py (hunk 1 above)
Comment (das-g) [inline osmaxx/excerptexport/forms/existing_form.py]: Yes, but that's orthogonal to this PR. Will be handled in #406.

**Pull request merged**